lib/osf-components/addon/components/sign-up-form/template.hbs

```diff
@@ -3,44 +3,44 @@
-        model=this.userRegistration
-        valuePath='fullName'
-        shouldShowMessages=this.didValidate
-        disabled=this.hasSubmitted
-        placeholder=(t 'osf-components.sign-up-form.full_name')
-        ariaLabel=(t 'osf-components.sign-up-form.full_name')
-    }}
-    {{validated-input/text
-        (html-attributes data-test-sign-up-email-1=true)
-        model=this.userRegistration
-        valuePath='email1'
-        shouldShowMessages=this.didValidate
-        disabled=this.hasSubmitted
-        placeholder=(t 'osf-components.sign-up-form.contact_email')
-        ariaLabel=(t 'osf-components.sign-up-form.contact_email')
-    }}
-    {{validated-input/text
-        (html-attributes data-test-sign-up-email-2=true)
-        model=this.userRegistration
-        valuePath='email2'
-        shouldShowMessages=this.didValidate
-        disabled=this.hasSubmitted
-        placeholder=(t 'osf-components.sign-up-form.confirm_email')
-        ariaLabel=(t 'osf-components.sign-up-form.confirm_email')
-    }}
-    {{validated-input/text
-        (html-attributes data-test-sign-up-password=true)
-        password=true
-        model=this.userRegistration
-        valuePath='password'
-        shouldShowMessages=this.didValidate
-        disabled=this.hasSubmitted
-        onKeyUp=(perform this.strength this.userRegistration.password)
-        placeholder=(t 'osf-components.sign-up-form.password')
-        ariaLabel=(t 'osf-components.sign-up-form.password')
-    }}
+    <ValidatedInput::Text
+        data-test-sign-up-full-name=true
+        @model={{this.userRegistration}}
+        @valuePath='fullName'
+        @shouldShowMessages={{this.didValidate}}
+        disabled={{this.hasSubmitted}}
+        placeholder={{t 'osf-components.sign-up-form.full_name'}}
+        ariaLabel={{t 'osf-components.sign-up-form.full_name'}}
+    />
+    <ValidatedInput::Text
+        data-test-sign-up-email-1=true
+        @model={{this.userRegistration}}
+        @valuePath='email1'
+        @shouldShowMessages={{this.didValidate}}
+        disabled={{this.hasSubmitted}}
+        placeholder={{t 'osf-components.sign-up-form.contact_email'}}
+        ariaLabel={{t 'osf-components.sign-up-form.contact_email'}}
+    />
+    <ValidatedInput::Text
+        data-test-sign-up-email-2=true
+        @model={{this.userRegistration}}
+        @valuePath='email2'
+        @shouldShowMessages={{this.didValidate}}
+        disabled={{this.hasSubmitted}}
+        placeholder={{t 'osf-components.sign-up-form.confirm_email'}}
+        ariaLabel={{t 'osf-components.sign-up-form.confirm_email'}}
+    />
+    <ValidatedInput::Text
+        data-test-sign-up-password=true
+        password={{true}}
+        @model={{this.userRegistration}}
+        @valuePath='password'
+        @shouldShowMessages={{this.didValidate}}
+        disabled={{this.hasSubmitted}}
+        {{on 'keyup' (perform this.strength this.userRegistration.password)}}
```

Comment:
Should we pass `(perform this.strength this.userRegistration.password` as an argument tot the `@onKeyUp` parameter since the [`<ValidatedInput::Text>` component actually applies `@onKeyUp` in a modifier](https://github.com/CenterForOpenScience/ember-osf-web/blob/basket/barium/lib/osf-components/addon/components/validated-input/text/template.hbs#L22)?

Reply:
So you're suggesting we instead `@onKeyUp={{perform this.strength this.userRegistration.password}}`?